src/BurgerIcon.js

```diff
@@ -40,11 +40,23 @@ let BurgerIcon = React.createClass({
       outline: 'none'
     });
 
+    var containerStyle = appendVendorPrefix({
+      position: "absolute",
+      left: "10%",
+      right: "10%",
+      top: "10%",
+      bottom : "10%",
+      margin: 0,
+      padding: 0
+    });
+
     return (
-      <div>
-        <span className="bm-burger-icon" style={ this.getLineStyle(0) }></span>
-        <span className="bm-burger-icon" style={ this.getLineStyle(1) }></span>
-        <span className="bm-burger-icon" style={ this.getLineStyle(2) }></span>
+      <div className="bm-burger-button">
+        <div className="bm-burger-icon-container" style={ containerStyle }>
```

Comment:
I don't think this container is needed - if its only purpose is to create the 10% gap around the `span` elements, I think the button works just fine without that. The effect created by the size of the button before wasn't particularly deliberate :)